.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 out
+usage.ts
```

Comment:
Why not commit the file?

Reply:
I just use it as a playground, it's not meant to be an actual example file, so it may be misleading